### ISSUE TITLE
Trailing comma after `db` statement lacks helpful error

### ISSUE THREAD

**Issue by bropedio** [RPGHacker/asar#290]
Apologies if there is a better place for me to report this, but I ran into the following nonspecific error when compiling today (with verbose option)

```
Asar 1.90pre, originally developed by Alcaro, maintained by Asar devs.
Source code: https://github.com/RPGHacker/asar
Errors were detected while assembling the patch. Assembling aborted. Your ROM has not been modified.
Asar 1.90pre, originally developed by Alcaro, maintained by Asar devs.
Source code: https://github.com/RPGHacker/asar

Assembling completed without problems.
```
The ROM was not assembled, despite the final log output. The error turned out to be a trailing comma after a `db` statement.
```
  dw $812B : db "Attack",$00,
```

**Comment by p4plus2** [comment]
We've been unable to replicate the issue.  Can you provide additional context?

If you're using a batch script have you tried manually patching?  It seems like the contents of stderr are being masked.

**Comment by Alcaro** [comment]
Reduced testcase to
\<trillian\> !rb as2 -b master db test:
\<randombot999\> Errors occurred while assembling.

Nothing to do with the comma.

(Doesn't even throw a phantom error, which may be a separate bug...)

**Comment by randomdude999** [comment]
fixed in 4cfb07217a38e8decd60ee72c476d529713340a7
